build.gradle

```diff
@@ -72,6 +73,9 @@ allprojects {
     }
 
     dependencies {
+        // apiguardian-api dependency added to remove "warning: unknown enum constant Status.STABLE" (used by junit-jupiter 5.8.2)
+        implementation "org.apiguardian:apiguardian-api:${apiguardianVersion}"
```

Comment:
As this is related to test-compilation and only to "statistik-service", I think we should move it to that module and use testImplementation instead.